pip_compile/scripts/pip-compile-container.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env sh
+set -e
+
+# Docker Arguments
+IMAGE_REF="$1"
+IMAGE_CONTEXT="$2"
+DOCKERFILE="$3"
+IMAGE_TARGET="$4"
+BUILD_TAG="$IMAGE_REF:build"
+
+# Requirements Paths
+LOCAL_REQ_PATH="$5"
+REQ_MOUNT_PATH="/.pip-requirements/mount/"
+
+# Script Paths
+LOCAL_SCRIPT_PATH=$(dirname "$0")
+REQ_SCRIPT_PATH="/.pip-requirements/scripts/"
+
+# Build image.
+echo "Please wait: Building Docker image..."
+docker build --quiet \
+  --tag "$BUILD_TAG" \
+  --target "$IMAGE_TARGET" \
+  --cache-from "$IMAGE_REF" \
+  --file "$DOCKERFILE" "$IMAGE_CONTEXT"
+
+# Compile pip requirements.
+echo "Please wait: Compiling pip requirements with pip-tools..."
+docker run -i --rm \
```

Comment:
If this fails for whatever reason, should the user get a nice message telling them that the `pip-compile` operation wasn't successful? Right now if pip dies, a stack trace is printed and then nothing else.

Same for the `docker build` step.